app/controllers/CreditAndRefundController.scala

```diff
@@ -88,7 +91,7 @@ class CreditAndRefundController @Inject()(val authorisedFunctions: FrontendAutho
         )
 
         val creditAndRefundType: Option[UnallocatedCreditType] = maybeUnallocatedCreditType(credits, balance, isMFACreditsAndDebitsEnabled, isCutOverCreditsEnabled)
-
+        auditClaimARefund(balance, credits)
```

Comment:
Feature Switch: This TxM event needs to be behind a feature switch - R7c Viewer TXM Events